lib/cli.js

```diff
@@ -157,6 +157,21 @@ function runDevelopmentMode (testModulePath) {
     callback()
   })
 
+  vorpal.command('runto <stepNumber>')
+  .description('Run from current step to step number')
+  .action(function (args, callback) {
+    handleRun(co(function* () {
+      while (!(tester.getCurrentStepNumber() === String(args.stepNumber) || tester.isDone())) {
```

Comment:
Another thought:

What if we make it an option for the `continue` command?

e.g.

```
continue --until 5
```

Or in short

```
c -u 5
```